src/components/Gather/Gather.js

```diff
@@ -1,49 +1,27 @@
 import React, { Component } from 'react';
-import { View, Text } from 'react-native';
-import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-import TextStyles from '../../helpers/TextStyles';
-import strings from '../../localization';
-import getUser from '../../selectors/UserSelector';
-import getRole from '../../selectors/RoleSelector';
-import styles from './styles';
+import { StyleSheet, View } from 'react-native';
+import Mapbox from '@mapbox/react-native-mapbox-gl';
 
-class Gather extends Component {
-  static navigatorStyle = {
-    navBarHidden: true,
-  };
+Mapbox.setAccessToken('pk.eyJ1IjoicXFtZWxvIiwiYSI6ImNqbWlhOXh2eDAwMHMzcm1tNW1veDNmODYifQ.vOmFAXiikWFJKh3DpmsPDA');
 
+export default class App extends Component {
   render() {
-    const { user } = this.props;
-    const { role } = this.props;
     return (
-      <View style={styles.container}>
-        <Text style={TextStyles.lightTitle}>{strings.gatherTitle}</Text>
-        <Text>{`${strings.homeMessage} ${user && user.name}`}</Text>
-        <Text style={TextStyles.lightTitle}>{`${role}`}</Text>
+      <View style={stylesGather.container}>
+        <Mapbox.MapView
+            styleURL={Mapbox.StyleURL.Street}
+            zoomLevel={15}
+            centerCoordinate={[11.256, 43.770]}
+            style={styles.container}>
```

Comment:
If you delete the styles import  when the user go to the gather page the app display an error saying that doesn't find the styles variable. 

Fixing that import when I go to the gather page I only see a green screen without the map.

Reply:
Should be fixed in next commit, its about to get pushed